scripts/trilinos-integration/README.md

```diff
@@ -0,0 +1,59 @@
+![Kokkos](https://avatars2.githubusercontent.com/u/10199860?s=200&v=4)
+
+# Kokkos: Promotion Test Debugging
+
+This explains the use (and basic implementation details) of promotion testing
+new Kokkos and Kokkos Kernels branches. We first introduce a test for first
+validating a "clean" Trilinos branch that all tests should be passing on.
```

Comment:
Not sure what you mean by "Trilinos branch".

Reply:
Tried to add clarifying text